Sawa 207 migrate achievements

fabiogiorgio opened the pull request:
This PR contains:
> Achievements
> The fix for the issue about users having an empty array of Achievements
> Shower Data
> Terms and Conditions
> Disclaimers

fabiogiorgio committed:
commit wip still have an issue about using User = ShowerShow.Models.User;

fabiogiorgio committed:
send help

sos

fabiogiorgio committed:
achievements fucking done

adios

fabiogiorgio committed:
user now has achievements

fabiogiorgio committed:
from laptop to pc

we

fabiogiorgio committed:
terms and condition done

from laptop

fabiogiorgio committed:
discliamers and terms and conditions

from laptop to pc

fabiogiorgio committed:
idk

fuck gitignore

egecnr commented on ShowerShow/Control/DisclaimersController.cs:
```diff
@@ -0,0 +1,62 @@
+namespace ShowerShow.Control
+{
+    public class DisclaimersController
+    {
```
Why are disclaimers  not in extra function ?? It is in the wrong function app. Can you move these into extra function instead?

egecnr commented on ShowerShow/Control/TermsAndConditionsController.cs:
```diff
@@ -0,0 +1,65 @@
+
+namespace ShowerShow.Control
+{
+    public class TermsAndConditionsController
```
Same here... These endpoints belong to the extra function not to showershow one

egecnr commented on ShowerShow/DTO/UpdateDisclaimerDTO.cs:
```diff
@@ -0,0 +1,13 @@
+{
+    internal class UpdateDisclaimerDTO
+    {
+        public enum AvailableDisclaimers { get; set; }
```
The app has a syntax error because of this so i was not even able to test the app

egecnr commented on ShowerShow/Repository/Interface/IDisclaimersRepository.cs:
```diff
@@ -0,0 +1,15 @@
+{
+    public class IDisclaimerRepository
+    {
+        public Task<Disclaimers> GetDisclaimres(Disclaimers disclaimers);
```
Everything dislaimer and term condition related should move to the other function

egecnr reviewed (changes requested):
Disclaimer and terms and conditions are in the wrong place you have to move them  to the extra function. Also your app doesnt run so i was not able to test it

fabiogiorgio committed:
moved disclaimers-terms and condition to extra function

fabiogiorgio committed:
during this commit I fixed 150 errors

fuck me

fabiogiorgio committed:
auth ege part FABIO NEEDS COMMENT CHECKING CHECK COMMENTS FABIO

HOHO

fabiogiorgio committed:
cosmin

fabiogiorgio committed:
sos

fabiogiorgio committed:
work done at school

fixed terms and conditions
fixed disclaimers
manually created each of them in the db and then tested the get endpoint. it work

fabiogiorgio committed:
removed security

 [ExampleAuth] - done

fabiogiorgio committed:
removed security

 [ExampleAuth]

fabiogiorgio committed:
last changes fabio

egecnr committed:
Fixed this app

egecnr committed:
Merge remote-tracking branch 'origin/dev' into SAWA-207-MigrateAchievements

# Conflicts:
#	ExtraFunction/Program.cs
#	ExtraFunction/bin/Debug/net6.0/.azurefunctions/Microsoft.Azure.WebJobs.Extensions.FunctionMetadataLoader.dll
#	ExtraFunction/bin/Debug/net6.0/Microsoft.Azure.Functions.Worker.Core.dll
#	ExtraFunction/bin/Debug/net6.0/Microsoft.IdentityModel.JsonWebTokens.dll
#	ExtraFunction/bin/Debug/net6.0/Microsoft.IdentityModel.Logging.dll
#	ExtraFunction/bin/Debug/net6.0/Microsoft.IdentityModel.Tokens.dll
#	ExtraFunction/bin/Debug/net6.0/System.IdentityModel.Tokens.Jwt.dll
#	ExtraFunction/obj/Debug/net6.0/ExtraFunction.csproj.AssemblyReference.cache
#	Function2/obj/project.nuget.cache
#	ShowerShow/DAL/DatabaseContext.cs
#	ShowerShow/Model/User.cs
#	ShowerShow/Program.cs
#	ShowerShow/bin/Debug/net6.0/.azurefunctions/Microsoft.Azure.Functions.Worker.Extensions.dll
#	ShowerShow/bin/Debug/net6.0/.azurefunctions/Microsoft.Azure.Functions.Worker.Extensions.pdb
#	ShowerShow/bin/Debug/net6.0/Azure.Core.dll
#	ShowerShow/bin/Debug/net6.0/Microsoft.Bcl.AsyncInterfaces.dll
#	ShowerShow/bin/Debug/net6.0/Microsoft.WindowsAzure.Storage.dll
#	ShowerShow/bin/Debug/net6.0/ShowerShow.deps.json
#	ShowerShow/bin/Debug/net6.0/ShowerShow.dll
#	ShowerShow/bin/Debug/net6.0/ShowerShow.exe
#	ShowerShow/bin/Debug/net6.0/ShowerShow.pdb
#	ShowerShow/bin/Debug/net6.0/functions.metadata
#	ShowerShow/bin/Debug/net6.0/local.settings.json
#	ShowerShow/local.settings.json
#	ShowerShow/obj/Debug/net6.0/ShowerShow.csproj.AssemblyReference.cache
#	ShowerShow/obj/Debug/net6.0/ShowerShow.csproj.CoreCompileInputs.cache
#	ShowerShow/obj/Debug/net6.0/ShowerShow.csproj.FileListAbsolute.txt
#	ShowerShow/obj/Debug/net6.0/ShowerShow.dll
#	ShowerShow/obj/Debug/net6.0/ShowerShow.genruntimeconfig.cache
#	ShowerShow/obj/Debug/net6.0/ShowerShow.pdb
#	ShowerShow/obj/Debug/net6.0/ref/ShowerShow.dll
#	ShowerShow/obj/Debug/net6.0/refint/ShowerShow.dll
